### PR TITLE
feat: add pt-pt translations

### PR DESCRIPTION
**Pull request by PedroMiguelSS**
This PR adds Portuguese translations.

### CODE REVIEW

File: intl/messages/pt.json [hunk 1]
```diff
@@ -1,149 +1,134 @@
...
-        "title": "Use PubSub to communicate in realtime with the other peers",
-        "desc": "IPFS nodes can create network topologies based on topics of interest to broadcast events in realtime."
+        "title": "Usa PubSub para comunicar em tempo real com todos os outros peers",
+        "desc": "Os nós do IPFS criam topologias de rede dedicadas à volta dos tópicos de interesse de forma a que os eventos sejam propagados em tempo real."
       },
       {
-        "title": "Add and retrieve files from anywhere in the IPFS Network",
-        "desc": "IPFS is designed to use the power of Content Addressing and Process Addressing to find the nodes in the network that have the content you are looking for and retrieve it. In the same way, you can add any piece of data and other nodes will find it too."
+        "title": "Providencia e/ou encontra ficheiros de qualquer parte da rede IPFS",
+        "desc": "IPFS foi desenhado para tirar proveito de endereçamento de conteúdo e processo para encontrar nós na rede com os dados dos quais procuras. Da mesma forma, todos os dados que forem adicionados também vão estar disponíveis para qualquer outro nó."
```
Comment (fsdiogo) [inline intl/messages/pt.json]: IPFS foi desenhado para tirar proveito `do` endereçamento de conteúdo e ~processo~ para encontrar nós na rede com os dados dos quais procuras.

File: intl/messages/pt.json [hunk 2]
```diff
@@ -1,149 +1,134 @@
...
-        "title": "Use the DAG API to traverse over any hash linked data structure",
-        "desc": "IPFS uses [IPLD](https://ipld.io/), the InterPlanetary Linked-Data, a model that enables you interact with data from multiple sources, such as git, blockchains and more."
+        "title": "Usa a DAG API para atravessar qualquer estrutura de dados que seja ligada por hashes",
+        "desc": "IPFS usa o [IPLD](https://ipld.io/), a \"InterPlanetary Linked-Data\", um modelo de dados que permite interagir com dados de muitas fontes, como por exemplo git, blockchains entre outras."
       },
       {
-        "title": "libp2p is built in",
-        "desc": "[libp2p](https://libp2p.io/) is the Modular Networking Stack that was created for IPFS and now you can use it through IPFS for your application directly as well."
+        "title": "libp2p incluída",
+        "desc": "libp2p é o stack de rede modular que foi criado para o IPFS e agora podes usar na tua aplicação através do IPFS."
```
Comment (fsdiogo) [inline intl/messages/pt.json]: libp2p é `a` stack de rede modular que foi criado para o IPFS e agora podes usar `também directamente` na tua aplicação.

File: intl/messages/pt.json [hunk 3]
```diff
@@ -1,149 +1,134 @@
...
       },
       {
-        "title": "Run it as a daemon",
-        "desc": "js-ipfs also comes with the option to run it as a daemon so that you can interact with it using the HTTP API that you are familiar with from go-ipfs."
+        "title": "Executa como um processo independente",
+        "desc": "js-ipfs vem também com a opção de ser executado como um processo independente que podes intereagir através da API HTTP que já estás famialirizado do go-ipfs."
```
Comment (fsdiogo) [inline intl/messages/pt.json]: js-ipfs vem também com a opção de ser executado como um processo independente `com o qual` podes `interagir` através da API HTTP que já estás `familiarizado` do go-ipfs.

File: intl/messages/pt.json [hunk 4]
```diff
@@ -1,149 +1,134 @@
...
       },
       {
-        "title": "Create stable addresses for changing data",
-        "desc": "It comes with IPNS, the Interplanetary Naming System, a way to author mutable pointers (records) to distribute updates in an authenticated and certified way."
+        "title": "Cria endereços estáveis para os dados em constante mudança",
+        "desc": "O IPNS, \"Interplanetary Naming System\", oferece uma forma de criar ponteiros mutáveis (registos) para propagar actualizações de uma forma certificada."
```
Comment (fsdiogo) [inline intl/messages/pt.json]: `atualizações`

File: intl/messages/pt.json [hunk 5]
```diff
@@ -1,149 +1,134 @@
...
+    "toggleText": "Ativar Service Worker",
+    "activatingToggleText": "A ativar o Service Worker...",
+    "activationErrorMessage": "Ocorreu um problema ao ativar o Service Worker.",
+    "deactivationErrorMessage": "Ocorreu um problema ao desativar o Service Worker.",
+    "nonRootScopeWarningMessage": "O Service Worker não pode ser ativo neste URL. Tenta em https://js.ipfs.io.",
```
Comment (fsdiogo) [inline intl/messages/pt.json]: ativo -> ativado

**Comment by lidel** [comment]
Does this PR supersede #102  ? 

FYSA we are moving to Transifex as the source of truth for translations.
We already got French translations finished that way, would be good to switch to that asap.

- https://github.com/ipfs/i18n
- https://www.transifex.com/ipfs/js-ipfs-website 
  (set up to push new strings from `intl/messages/en.json` twice a day)


PS. Are you able to review and merge #168 ?

**Comment by PedroMiguelSS** [comment]
Yes, this one supersedes #102 and I'll close it. This PR is the valid one for PT translation.

Cool! I did request access on https://www.transifex.com/ipfs/js-ipfs-website/ to join the team. I'm currently waiting for the access. When I get it I'll update PT locale there and follow your instructions.

On my way to review #168.

**Comment by lidel** [comment]
@PedroMiguelSS done! 

You should be able to fetch translations via `tx pull -a` now

Note: only 100% translations will be fetched:
https://github.com/ipfs/js.ipfs.io/blob/1d1cfe6ce6ff57a1612393248a022777c375dd10/.tx/config#L6

Check notes at https://github.com/ipfs/i18n#transifex-101
In case you have any questions I'll be happy to help :)

**Pull request merged**